### ISSUE TITLE
Can't Discover Broadlink RM Pro +    HA Bridge 5.3.0

### ISSUE THREAD

**Issue by homeautomate8** [bwssytems/ha-bridge#1154]
Have the above on 192.168.0.134 port 80 and the HA Bridge on 192.168.0.38 port 8083. It's not discovering the Broadlink device. Log as follows

11-19-2019 22:47:18.080 | INFO | Broadlink discover.... | com.bwssystems.HABridge.plugins.broadlink.BroadlinkHome
11-19-2019 22:47:19.110 | INFO | The requested route [/description.xml] has not been mapped in Spark for Accept: [application/json, text/plain, */*] | spark.http.matching.MatcherFilter
11-19-2019 22:47:21.134 | INFO | The requested route [/description.xml] has not been mapped in Spark for Accept: [application/json, text/plain, */*] | spark.http.matching.MatcherFilter
11-19-2019 22:47:23.108 | WARN | Did not discover any Broadlinks.
11-19-2019 22:47:23.109 | INFO | Broadlink Home created. | com.bwssystems.HABridge.plugins.broadlink.BroadlinkHome

Tried all the obvious so far. Vera controller discovered no problem.

**Comment by homeautomate8** [comment]
Ok discovered this issue closed with no apparent solution https://github.com/bwssytems/ha-bridge/issues/984 As it was closed am I to assume that the poster solved their problem. Turning on WiFi which is a no brainer and setting static ip address for the RM made absolutely no difference. The question now is does the HA Bridge support the Broadlink RM Pro +. It has no problem learning RF and IR codes.

**Comment by homeautomate8** [comment]
Contents of the service daemon. Couldn't find Broadlink. Looks like Discovery service is not working. I changed the Bridge to port 80 but it still makes no difference

ha-bridge.service
   Loaded: loaded (/etc/systemd/system/ha-bridge.service; enabled; vendor preset: enabled)
   Active: active (running) since Sat 2019-11-23 18:26:00 GMT; 3h 7min ago
 Main PID: 896 (java)
    Tasks: 24 (limit: 2200)
   Memory: 39.4M
   CGroup: /system.slice/ha-bridge.service
           └─896 /usr/bin/java -jar -Dconfig.file=/home/pi/ha-bridge/data/habridge.config /home/pi/ha-bridge/ha-bridge-5.3.0.jar -Dserver.port=80

Nov 23 18:26:08 raspberrypi java[896]: 2019-11-23 18:26:08,282 [main] WARN  com.bwssystems.HABridge.plugins.broadlink.BroadlinkHome - Did not discover any Broadlinks.
Nov 23 18:26:08 raspberrypi java[896]: 2019-11-23 18:26:08,283 [main] INFO  com.bwssystems.HABridge.plugins.broadlink.BroadlinkHome - Broadlink Home created.
Nov 23 18:26:08 raspberrypi java[896]: 2019-11-23 18:26:08,290 [main] INFO  com.bwssystems.HABridge.plugins.moziot.MozIotHome - Mozilla IOT Home created. No Mozilla IOTs configured.
Nov 23 18:26:08 raspberrypi java[896]: 2019-11-23 18:26:08,421 [main] INFO  com.bwssystems.HABridge.devicemanagmeent.DeviceResource - HABridge device management service started....
Nov 23 18:26:08 raspberrypi java[896]: 2019-11-23 18:26:08,533 [main] INFO  com.bwssystems.HABridge.hue.HueMulator - Hue emulator service started....
Nov 23 18:26:08 raspberrypi java[896]: 2019-11-23 18:26:08,646 [main] INFO  com.bwssystems.HABridge.upnp.UpnpSettingsResource - Description xml service started....
Nov 23 18:26:08 raspberrypi java[896]: 2019-11-23 18:26:08,664 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - UPNP Discovery Listener starting....
Nov 23 18:26:08 raspberrypi java[896]: 2019-11-23 18:26:08,666 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - Create and run mDNS service.
Nov 23 18:26:08 raspberrypi java[896]: 2019-11-23 18:26:08,758 [main] INFO  com.bwssystems.HABridge.upnp.UpnpListener - UPNP Discovery Listener running and ready....

**Comment by homeautomate8** [comment]
OK I'm assuming HA Bridge does not support my particular model of Broadlink RM Pro +
For anyone else experiencing similar difficulties my model details as follows
Broadlink RM Pro + IR+RF+WiFi
S/N 33222***************
Firmware v52

If anyone does indeed does find a solution I'll leave this issue open and maybe you could post the details.

**Comment by Nachtfalke49** [comment]
Hi,

I have the same Problem:

Broadlink RM Pro + IR+RF+WiFi EU Model
Firmware: 57

12-14-2019 11:14:00.294	INFO	Broadlink discover....	com.bwssystems.HABridge.plugins.broadlink.BroadlinkHome
12-14-2019 11:14:05.307	WARN	Did not discover any Broadlinks.	com.bwssystems.HABridge.plugins.broadlink.BroadlinkHome

**Comment by polarbeared** [comment]
Yep, me too. Same device (RM Pro) with foxes IP. Port scan doesn’t show open ports for RM Pro, but shows 80 for Bridge. Don’t understand what’s going on.

**Comment by Nachtfalke49** [comment]
I have the same problem.

**Comment by alpet83** [comment]
Same problem I solved, via change code in broadlink_cli: 
-parser.add_argument("data", nargs='*', help="Data to send or convert") 
+parser.add_argument("--data", nargs='*', help="Data to send or convert")
Now it works perfectly: broadlink_cli --device @ac --send -data @t24

**Comment by mralapete** [comment]
Same problem I solved, via change code in broadlink_cli:
-parser.add_argument("data", nargs='', help="Data to send or convert")
+parser.add_argument("--data", nargs='', help="Data to send or convert")
Now it works perfectly: broadlink_cli --device @ac --send -data @t24

Just received this email stating that the user had found the following solution but it’s not appearing here. Is it spam or factual.

**Comment by alpet83** [comment]
> Just received this email stating that the user had found the following solution but it’s not appearing here. Is it spam or factual.

It's my mistake, just messed thread https://github.com/mjg59/python-broadlink/issues/194

**Comment by mralapete** [comment]
> > Just received this email stating that the user had found the following solution but it’s not appearing here. Is it spam or factual.
> 
> It's my mistake, just messed thread [mjg59/python-broadlink#194](https://github.com/mjg59/python-broadlink/issues/194)

Ok. So you posted a solution to a different issue here in error. Understood.
